src/bin/cli/kedqr/hash.rs

```diff
@@ -0,0 +1,70 @@
+use catalyst_toolbox::kedqr::generate;
+use catalyst_toolbox::kedqr::QrPin;
+use chain_crypto::bech32::Bech32;
+use chain_crypto::{Ed25519Extended, SecretKey};
+use std::fs::File;
+use std::io::Write;
+use std::{
+    error::Error,
+    fs::OpenOptions,
+    io::{BufRead, BufReader},
+    path::PathBuf,
+};
+use structopt::StructOpt;
+
+/// QCode CLI toolkit
+#[derive(Debug, PartialEq, StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct QrCodeHashCmd {
+    /// Path to file containing ed25519extended bech32 value.
+    #[structopt(short, long, parse(from_os_str))]
+    input: PathBuf,
```

Comment:
these arguments seems to be the same than for the `QrCodeHashImg`, should we use [this](https://docs.rs/structopt/0.3.23/structopt/#flattening) to avoid duplication?